pep-0630.rst

```diff
@@ -1,43 +1,44 @@
 PEP: 630
 Title: Isolating Extension Modules
 Author: Petr Viktorin <encukou@gmail.com>
-Discussions-To: capi-sig@python.org
+Discussions-To: https://mail.python.org/archives/list/capi-sig@python.org/
```

Comment:
```suggestion
Discussions-To: capi-sig@python.org
```

#2467 will automatically convert bar email addresses to a direct link to the list, so this change is no longer necessary